server/consts_p.go

```diff
@@ -20,4 +20,7 @@ const (
 
 	ReportMigrationFrequency        = 1 * time.Hour
 	WattTimeWeekDataUpdateFrequency = 24 * time.Hour
+
+	apiArchiveLimit = 1
+	apiArchiveRate  = time.Second
```

Comment:
Is 1 ever a good choice for a ratelimit?  Maybe it should be more like 3 every 3 seconds.

Reply:
Agree